CMakeLists.txt

```diff
@@ -45,6 +45,18 @@ elseif(COVERAGE)
 endif()
 unset(COVERAGE CACHE)
 
+# LLVM_COVERAGE ==> LIBCELLML_LLVM_COVERAGE
+set(_PARAM_ANNOTATION "Enable OSX coverage testing.")
```

Comment:
Why OSX? I would use macOS. Also, why only have LLVM-based coverage on macOS? Surely, it should work on all platforms?

Reply:
Search and replace has failed me :(